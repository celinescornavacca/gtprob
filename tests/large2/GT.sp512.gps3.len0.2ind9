(((((EO1,EO2),EO0),((FO1,FO2),FO0)),((EA0,(EA1,EA2)),((((((DI1,DI2),DI0),(((YR1,YR2),YR0),((ZR1,ZR2),ZR0))),(FE0,(FE1,FE2))),(((YB1,YB2),YB0),((ZB1,ZB2),ZB0))),(((SR0,(SR1,SR2)),((TR1,TR2),TR0)),(((UM0,(UM1,UM2)),(VM0,(VM1,VM2))),((HF1,HF2),HF0)))))),(((((((((((QA1,QA2),QA0),(((BH1,BH2),BH0),((WK0,(WK1,WK2)),(XK0,(XK1,XK2))))),((((((SM1,SM2),SM0),(TM0,(TM1,TM2))),((PI0,(PI1,PI2)),(((UR1,UR2),UR0),((VR1,VR2),VR0)))),((XB1,XB2),XB0)),(((((SK0,(SK1,SK2)),(TK0,(TK1,TK2))),(((JO1,JO2),JO0),((HT0,(HT1,HT2)),(IT0,(IT1,IT2))))),(((SG1,SG2),SG0),((TG1,TG2),TG0))),((ZD0,(ZD1,ZD2)),((AE1,AE2),AE0))))),(((KJ1,KJ2),KJ0),(LJ0,(LJ1,LJ2)))),(((((NQ0,(NQ1,NQ2)),(OQ0,(OQ1,OQ2))),((DM1,DM2),DM0)),(((AD1,AD2),AD0),((BD1,BD2),BD0))),(((((ZO1,ZO2),ZO0),((AR0,(AR1,AR2)),((BR1,BR2),BR0))),(GH0,(GH1,GH2))),((BC0,(BC1,BC2)),(CC0,(CC1,CC2)))))),(((((QB1,QB2),QB0),((RB1,RB2),RB0)),((UD0,(UD1,UD2)),((CK0,(CK1,CK2)),(((YK1,YK2),YK0),((ZK1,ZK2),ZK0))))),(((DA0,(DA1,DA2)),(((VJ0,(VJ1,VJ2)),(((QL1,QL2),QL0),(((UP0,(UP1,UP2)),((VP1,VP2),VP0)),(TL0,(TL1,TL2))))),((((ZF1,ZF2),ZF0),((((FT1,FT2),FT0),((GT1,GT2),GT0)),((GM1,GM2),GM0))),((DC1,DC2),DC0)))),((((AC1,AC2),AC0),((((VL0,(VL1,VL2)),(WL0,(WL1,WL2))),(FG0,(FG1,FG2))),(((OM0,(OM1,OM2)),((PM1,PM2),PM0)),((FM1,FM2),FM0)))),((XA1,XA2),XA0))))),(((((((OJ1,OJ2),OJ0),(PJ0,(PJ1,PJ2))),((((UF0,(UF1,UF2)),(VF0,(VF1,VF2))),((PF0,(PF1,PF2)),(QF0,(QF1,QF2)))),((HE0,(HE1,HE2)),(IE0,(IE1,IE2))))),((BB0,(BB1,BB2)),((((((JD1,JD2),JD0),(((UI1,UI2),UI0),(((CO0,(CO1,CO2)),(DO0,(DO1,DO2))),(AO0,(AO1,AO2))))),(IC0,(IC1,IC2))),(((WQ1,WQ2),WQ0),((XQ1,XQ2),XQ0))),((((KT0,(KT1,KT2)),(LT0,(LT1,LT2))),(EM0,(EM1,EM2))),((KO0,(KO1,KO2)),(((FQ1,FQ2),FQ0),(GQ0,(GQ1,GQ2)))))))),(((((IS0,(IS1,IS2)),((PS0,(PS1,PS2)),((QS1,QS2),QS0))),((NN1,NN2),NN0)),(((EL1,EL2),EL0),((FL1,FL2),FL0))),((((ZE0,(ZE1,ZE2)),(((MM1,MM2),MM0),((NM1,NM2),NM0))),((KC1,KC2),KC0)),(((DK1,DK2),DK0),((ZM0,(ZM1,ZM2)),((AN1,AN2),AN0)))))),((((XI0,(XI1,XI2)),((((GS1,GS2),GS0),(HS0,(HS1,HS2))),((RQ1,RQ2),RQ0))),((EG1,EG2),EG0)),((GF1,GF2),GF0)))),((((VC1,VC2),VC0),(((((OT1,OT2),OT0),((PT1,PT2),PT0)),((NP1,NP2),NP0)),((CQ0,(CQ1,CQ2)),((DQ1,DQ2),DQ0)))),(((((((((RF1,RF2),RF0),((KG0,(KG1,KG2)),((LG1,LG2),LG0))),((((UQ0,(UQ1,UQ2)),(VQ0,(VQ1,VQ2))),((UL1,UL2),UL0)),(GJ0,(GJ1,GJ2)))),(((AL0,(AL1,AL2)),(BL0,(BL1,BL2))),((GG1,GG2),GG0))),(HD0,(HD1,HD2))),(((WC0,(WC1,WC2)),((((LK0,(LK1,LK2)),((MK1,MK2),MK0)),(LF0,(LF1,LF2))),(((XG1,XG2),XG0),(YG0,(YG1,YG2))))),(((GD1,GD2),GD0),((YJ0,(YJ1,YJ2)),(((OR1,OR2),OR0),((PR1,PR2),PR0)))))),((((EH1,EH2),EH0),(FH0,(FH1,FH2))),(((ES0,(ES1,ES2)),((FS1,FS2),FS0)),(((FF0,(FF1,FF2)),(((((ON1,ON2),ON0),(((NS1,NS2),NS0),((OS1,OS2),OS0))),(((EK1,EK2),EK0),(((IM1,IM2),IM0),(JM0,(JM1,JM2))))),(DG0,(DG1,DG2)))),(JC0,(JC1,JC2)))))),((UB0,(UB1,UB2)),(VB0,(VB1,VB2)))))),(((((((DD1,DD2),DD0),((ED1,ED2),ED0)),((((LC0,(LC1,LC2)),(MC0,(MC1,MC2))),((((WE0,(WE1,WE2)),(XE0,(XE1,XE2))),(GC0,(GC1,GC2))),((MA0,(MA1,MA2)),(((RC1,RC2),RC0),(((UC1,UC2),UC0),((WP0,(WP1,WP2)),(XP0,(XP1,XP2)))))))),(((SO0,(SO1,SO2)),(((AS1,AS2),AS0),((BS1,BS2),BS0))),((LL0,(LL1,LL2)),(ML0,(ML1,ML2)))))),(((((((PN1,PN2),PN0),(QN0,(QN1,QN2))),((PH1,PH2),PH0)),((((JS1,JS2),JS0),((KS1,KS2),KS0)),(TE0,(TE1,TE2)))),((ZA0,(ZA1,ZA2)),((((UK1,UK2),UK0),(VK0,(VK1,VK2))),((QE1,QE2),QE0)))),((((BM1,BM2),BM0),((CM1,CM2),CM0)),(XC0,(XC1,XC2))))),((FA0,(FA1,FA2)),(((CE1,CE2),CE0),((DE1,DE2),DE0)))),((((((((((LQ0,(LQ1,LQ2)),(MQ0,(MQ1,MQ2))),((JH0,(JH1,JH2)),((KH1,KH2),KH0))),((HI0,(HI1,HI2)),(II0,(II1,II2)))),(OD0,(OD1,OD2))),(((MD0,(MD1,MD2)),((ND1,ND2),ND0)),(((AG0,(AG1,AG2)),(BG0,(BG1,BG2))),((HG0,(HG1,HG2)),((UH0,(UH1,UH2)),((VH1,VH2),VH0)))))),(((((NO1,NO2),NO0),((OO1,OO2),OO0)),((LH1,LH2),LH0)),((((((((OP1,OP2),OP0),((PP1,PP2),PP0)),(PG0,(PG1,PG2))),(((((VO1,VO2),VO0),(WO0,(WO1,WO2))),((PL1,PL2),PL0)),((JP0,(JP1,JP2)),(KP0,(KP1,KP2))))),(CD0,(CD1,CD2))),((QD0,(QD1,QD2)),((((JK1,JK2),JK0),((KK1,KK2),KK0)),(((TS1,TS2),TS0),((US1,US2),US0))))),(((ZL0,(ZL1,ZL2)),(AM0,(AM1,AM2))),((CH0,(CH1,CH2)),((EI0,(EI1,EI2)),(FI0,(FI1,FI2)))))))),((((((((JE0,(JE1,JE2)),((KE1,KE2),KE0)),((PD0,(PD1,PD2)),(((NL1,NL2),NL0),((OL1,OL2),OL0)))),(((TB1,TB2),TB0),((BE0,(BE1,BE2)),(((PQ1,PQ2),PQ0),((QQ1,QQ2),QQ0))))),((VA0,(VA1,VA2)),(((KL0,(KL1,KL2)),(((AQ1,AQ2),AQ0),(BQ0,(BQ1,BQ2)))),(((EC1,EC2),EC0),(((SJ1,SJ2),SJ0),(TJ0,(TJ1,TJ2))))))),(((((NF0,(NF1,NF2)),(((GL0,(GL1,GL2)),(HL0,(HL1,HL2))),(DJ0,(DJ1,DJ2)))),((GI0,(GI1,GI2)),(((EJ1,EJ2),EJ0),((FJ1,FJ2),FJ0)))),((((XS1,XS2),XS0),((YS1,YS2),YS0)),(YH0,(YH1,YH2)))),(((QH1,QH2),QH0),((RH1,RH2),RH0)))),(((QT0,(QT1,QT2)),(RT0,(RT1,RT2))),((KD0,(KD1,KD2)),(LD0,(LD1,LD2))))),((((TC0,(TC1,TC2)),(((BJ1,BJ2),BJ0),(CJ0,(CJ1,CJ2)))),((EE0,(EE1,EE2)),((((SQ1,SQ2),SQ0),(TQ0,(TQ1,TQ2))),(HM0,(HM1,HM2))))),(((((QI0,(QI1,QI2)),(RI0,(RI1,RI2))),((FD1,FD2),FD0)),(((AB0,(AB1,AB2)),(((VI1,VI2),VI0),((WI1,WI2),WI0))),((((NC1,NC2),NC0),((OC1,OC2),OC0)),(DB0,(DB1,DB2))))),((BA1,BA2),BA0))))),((((BF1,BF2),BF0),((((TO1,TO2),TO0),(UO0,(UO1,UO2))),(((CL1,CL2),CL0),(DL0,(DL1,DL2))))),((((((((OA1,OA2),OA0),((SL0,(SL1,SL2)),((QO0,(QO1,QO2)),(RO0,(RO1,RO2))))),(((((FP1,FP2),FP0),(GP0,(GP1,GP2))),(((WM1,WM2),WM0),(XM0,(XM1,XM2)))),(PE0,(PE1,PE2)))),(((QP0,(QP1,QP2)),((RP1,RP2),RP0)),((BO1,BO2),BO0))),(((((KB1,KB2),KB0),((((XF0,(XF1,XF2)),(YF0,(YF1,YF2))),(((SF0,(SF1,SF2)),(TF0,(TF1,TF2))),((OF1,OF2),OF0))),(YE0,(YE1,YE2)))),((((LI1,LI2),LI0),(MI0,(MI1,MI2))),(ME0,(ME1,ME2)))),(((OG1,OG2),OG0),((SN0,(SN1,SN2)),(TN0,(TN1,TN2)))))),((((((((AI0,(AI1,AI2)),(((((MT1,MT2),MT0),(NT0,(NT1,NT2))),((JT1,JT2),JT0)),(BK0,(BK1,BK2)))),(((SH1,SH2),SH0),((TH1,TH2),TH0))),((WD0,(WD1,WD2)),(((QM1,QM2),QM0),((RM1,RM2),RM0)))),((((((BT0,(BT1,BT2)),(CT0,(CT1,CT2))),((EP1,EP2),EP0)),(NK0,(NK1,NK2))),(WF0,(WF1,WF2))),(((LE0,(LE1,LE2)),((ZH0,(ZH1,ZH2)),((XL0,(XL1,XL2)),((YL1,YL2),YL0)))),((NA1,NA2),NA0)))),(((((((MR1,MR2),MR0),((NR1,NR2),NR0)),((WG1,WG2),WG0)),((AF1,AF2),AF0)),(PB0,(PB1,PB2))),((GA1,GA2),GA0))),(CA0,(CA1,CA2))),((((((UN1,UN2),UN0),(VN0,(VN1,VN2))),((RK1,RK2),RK0)),((ER0,(ER1,ER2)),((FR1,FR2),FR0))),(((IB1,IB2),IB0),(((SI1,SI2),SI0),((IJ0,(IJ1,IJ2)),((JJ1,JJ2),JJ0))))))),((((NH1,NH2),NH0),((OH1,OH2),OH0)),(((FN0,(FN1,FN2)),(GN0,(GN1,GN2))),((QR0,(QR1,QR2)),((RR1,RR2),RR0))))))),((((((SA1,SA2),SA0),((LS0,(LS1,LS2)),((MS1,MS2),MS0))),(((((SC1,SC2),SC0),((((IK1,IK2),IK0),(((YQ1,YQ2),YQ0),((ZQ1,ZQ2),ZQ0))),(((OK1,OK2),OK0),(PK0,(PK1,PK2))))),((QG0,(QG1,QG2)),((RG1,RG2),RG0))),((YN0,(YN1,YN2)),(ZN0,(ZN1,ZN2))))),(((((((XD0,(XD1,XD2)),(YD0,(YD1,YD2))),((((MH1,MH2),MH0),((DT0,(DT1,DT2)),(ET0,(ET1,ET2)))),(((HH0,(HH1,HH2)),(IH0,(IH1,IH2))),(KF0,(KF1,KF2))))),(((((DF1,DF2),DF0),(EF0,(EF1,EF2))),((MB1,MB2),MB0)),((ZG0,(ZG1,ZG2)),((AH1,AH2),AH0)))),((((QK0,(QK1,QK2)),((VS0,(VS1,VS2)),((WS1,WS2),WS0))),(((((GO1,GO2),GO0),((RS0,(RS1,RS2)),(SS0,(SS1,SS2)))),(((EQ1,EQ2),EQ0),(((ZS1,ZS2),ZS0),(AT0,(AT1,AT2))))),((((IF1,IF2),IF0),((JF1,JF2),JF0)),((TI0,(TI1,TI2)),(((QJ1,QJ2),QJ0),((RJ1,RJ2),RJ0)))))),(((((((SD1,SD2),SD0),((TD1,TD2),TD0)),((ID1,ID2),ID0)),(((HC1,HC2),HC0),(((VD1,VD2),VD0),((GE0,(GE1,GE2)),((SP0,(SP1,SP2)),((TP1,TP2),TP0)))))),(JB0,(JB1,JB2))),((HB1,HB2),HB0)))),((((JN0,(JN1,JN2)),((KN1,KN2),KN0)),(((LO0,(LO1,LO2)),(MO0,(MO1,MO2))),((RL1,RL2),RL0))),((AP0,(AP1,AP2)),(BP0,(BP1,BP2))))),(((((WH0,(WH1,WH2)),((XH1,XH2),XH0)),(((MJ1,MJ2),MJ0),((NJ1,NJ2),NJ0))),(((BN0,(BN1,BN2)),((CN1,CN2),CN0)),((((((((LP1,LP2),LP0),(MP0,(MP1,MP2))),(AJ0,(AJ1,AJ2))),((OE1,OE2),OE0)),((NI0,(NI1,NI2)),(OI0,(OI1,OI2)))),(((KM1,KM2),KM0),(LM0,(LM1,LM2)))),(((((UG1,UG2),UG0),(VG0,(VG1,VG2))),((CG1,CG2),CG0)),((IA1,IA2),IA0))))),(((CB1,CB2),CB0),((YC0,(YC1,YC2)),((ZC1,ZC2),ZC0)))))),(((((RA0,(RA1,RA2)),(((DN1,DN2),DN0),((EN1,EN2),EN0))),(((NB0,(NB1,NB2)),((FK0,(FK1,FK2)),((GK1,GK2),GK0))),((PA1,PA2),PA0))),((((((RD1,RD2),RD0),(((HQ1,HQ2),HQ0),((IQ1,IQ2),IQ0))),((((RE1,RE2),RE0),(SE0,(SE1,SE2))),((((JI1,JI2),JI0),(KI0,(KI1,KI2))),((JQ0,(JQ1,JQ2)),(KQ0,(KQ1,KQ2)))))),(HA0,(HA1,HA2))),(((KA1,KA2),KA0),((LA1,LA2),LA0)))),((AA1,AA2),AA0)))))),(((((((((((AK1,AK2),AK0),((CP0,(CP1,CP2)),(DP0,(DP1,DP2)))),((DH1,DH2),DH0)),((IG0,(IG1,IG2)),((JG1,JG2),JG0))),((FC1,FC2),FC0)),(OB0,(OB1,OB2))),(((((GR1,GR2),GR0),((HR1,HR2),HR0)),(((((KR1,KR2),KR0),(LR0,(LR1,LR2))),(((WR0,(WR1,WR2)),(XR0,(XR1,XR2))),(PO0,(PO1,PO2)))),((UJ1,UJ2),UJ0))),(((((((BI1,BI2),BI0),(CI0,(CI1,CI2))),((CF1,CF2),CF0)),((YI0,(YI1,YI2)),((ZI1,ZI2),ZI0))),((((YP0,(YP1,YP2)),((ZP1,ZP2),ZP0)),(LN0,(LN1,LN2))),(TA0,(TA1,TA2)))),(((HP1,HP2),HP0),((IP1,IP2),IP0))))),((((MG1,MG2),MG0),(NG0,(NG1,NG2))),((((PC0,(PC1,PC2)),((QC1,QC2),QC0)),((LB1,LB2),LB0)),(((SB0,(SB1,SB2)),(((VE1,VE2),VE0),((HJ0,(HJ1,HJ2)),((HN0,(HN1,HN2)),((IN1,IN2),IN0))))),((YA1,YA2),YA0))))),((((JA1,JA2),JA0),(((UA1,UA2),UA0),((IL0,(IL1,IL2)),(JL0,(JL1,JL2))))),((((EB1,EB2),EB0),((FB1,FB2),FB0)),((((IR0,(IR1,IR2)),((JR1,JR2),JR0)),(((((XO1,XO2),XO0),((YO1,YO2),YO0)),((((HK1,HK2),HK0),(((CS1,CS2),CS0),(DS0,(DS1,DS2)))),(ZJ0,(ZJ1,ZJ2)))),(WB0,(WB1,WB2)))),(GB0,(GB1,GB2)))))),((((WA1,WA2),WA0),(((((WN1,WN2),WN0),((XN1,XN2),XN0)),((RN1,RN2),RN0)),(MN0,(MN1,MN2)))),(((NE0,(NE1,NE2)),(((YM1,YM2),YM0),((HO0,(HO1,HO2)),(IO0,(IO1,IO2))))),(((WJ0,(WJ1,WJ2)),(XJ0,(XJ1,XJ2))),(((((CR1,CR2),CR0),(DR0,(DR1,DR2))),((MF1,MF2),MF0)),(UE0,(UE1,UE2)))))))));
