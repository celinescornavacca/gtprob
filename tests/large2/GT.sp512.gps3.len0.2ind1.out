-2426.19476929
